app/scripts/dcmWalkingRetargetingXsens.xml

```diff
@@ -45,6 +45,19 @@
     <node>icub-virtualizer</node>
   </module>
 
+  <module>
+    <name>yarprobotinterface</name>
+    <parameters>--config TransformServer.xml</parameters>
+    <node>icub30</node>
+  </module>
+
+  <module>
+    <name>yarprobotinterface</name>
+    <parameters>--config RobotStateProvider_iCub.xml</parameters>
```

Comment:
Where `RobotStateProvider_iCub.xml` can be found? Is it provided by another repo? If yes you should add the dependences in the readme.